### ISSUE TITLE
PredeclaredId attibute woking but rubberduck PredeclaredId annotations are not

### ISSUE THREAD

**Issue by FullValueRider** [WaynePhillipsEA/twinbasic#126]
**Describe the bug**
Apologies if I am misunderstanding the current status.

The predeclaredId attribute is now working when I use

```
Attribute VB_PredeclaredId=True
```

but the RubberDuck/ Square bracket annotations do not work

```
'@PredeclaredId
[PredeclaredId]
```

**Comment by WaynePhillipsEA** [comment]
[PredeclaredId] appears to be working correctly.  Note that tB attributes need to appear immediately before the definition.

The rubberduck @PredeclaredId attribute is not working correctly since it wrongfully only being picked up before the class definition.  I'll have that fixed in the next update tomorrow.

**Comment by WaynePhillipsEA** [comment]
FIxed in v0.9.2699.  Thanks for reporting.
